cmd/checkmetrics/main.go

```diff
@@ -128,6 +133,45 @@ func processMetrics(context *cli.Context) (err error) {
 	return
 }
 
+
+// processMetrics generates a report using the metrics results
+// available in CSV files provided as input without use a basefile
+// as reference. This report shows basic statitistics just in
+// order to get an overview about the results.
+func processMetrics(context *cli.Context) (err error) {
+	var mtrdir string
+
+	mtrdir = context.GlobalString("metricsdir")
+	csvFiles, err := ioutil.ReadDir(mtrdir)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"NAME", "MEAN", "SD", "COV", "ITERS"})
+	for _, csvf := range csvFiles {
+		var thisCsv csvRecord
+		filepath := path.Join(mtrdir, csvf.Name())
+		err = thisCsv.load(filepath)
+		if  err != nil {
+			return err
+		}
+
+		row := []string{thisCsv.TestName,
+			strconv.FormatFloat(thisCsv.Mean, 'f', -1, 64),
```

Comment:
Might be nice to have the Min and Max here as well, as those are what we actually code back into the baseline toml files - what do you think?

Reply:
make sense, I will send a new version :)